types/utils.go

```diff
@@ -34,6 +36,17 @@ func MustSortJSON(toSortJSON []byte) []byte {
 	return js
 }
 
+// Formats a time.Time into a []byte that can be sorted
+func FormatTimeBytes(t time.Time) []byte {
+	return []byte(t.UTC().Round(0).Format("2006-01-02T15:04:05.000000000"))
```

Comment:
It seems like this format string should be a constant somewhere?

Reply:
Also what's with the choice of this specific day in 2006?

Reply:
@cwgoes https://pauladamsmith.com/blog/2011/05/go_time.html

Reply:
Huh interesting, did not know that!